app/src/test/java/org/oppia/android/app/testing/PlatformParameterIntegrationTest.kt

```diff
@@ -109,6 +91,7 @@ class PlatformParameterIntegrationTest {
   @Before
   fun setUp() {
     setUpTestApplicationComponent()
+    ShadowToast.reset()
```

Comment:
Why are we resetting at the start of the test? 

Reset method is called when we want to clear the recorded test, mostly after functions like `.showToast()` or after functions like where to count or read the toast text.

Reply:
![Screenshot 2021-07-22 at 18 55 00](https://user-images.githubusercontent.com/25057618/126646354-74dd9cc2-7496-4e88-a620-80ef3d35ec77.png)

Reply:
I was doing it just to clear any previous state. But yeah if we can work without it thats not an issue. Commiting this change

Reply:
I was doing it just to clear any previous state. But yeah if we can work without it thats not an issue. Commiting this change

Reply:
I don't know removing will work or not, I just shifted it to after. Let's see if it works, though moving it to after works, because let's say in future if we add more test with more toast text, it might fail.